pod/perlfunc.pod

```diff
@@ -9252,6 +9254,42 @@ The transliteration operator.  Same as
 L<C<yE<sol>E<sol>E<sol>>|/yE<sol>E<sol>E<sol>>.  See
 L<perlop/"Quote-Like Operators">.
 
+=item trim EXPR
+X<trim> X<remove_whitespace> (Perl 7+)
+
+=item trim
+
+=for Pod::Functions Strip whitespace from the beginning and end of a string.
+
+This function returns a string with whitespace stripped from the beginning
+and end of string. trim() will remove these characters:
+
+" ", an ordinary space.
+
+"\t", a tab.
+
+"\n", a new line (line feed).
+
+"\r", a carriage return.
+
+and other unicode characters that are flagged as whitespace.
+
+  $var = "  Hello world   ";  # "Hello world"
+  $var = "\t\t\tHello world"; # "Hello world"
+  $var = "Hello world\n";     # "Hello world"
+
+Trim is equivalent to:
+
+  $str =~ s/\A\s+|\s+\z//urg;
+
+L<C<trim>|/trim EXPR> is available only if the trim feature> is enabled
+or if it is prefixed with C<CORE::>.  The trim function is enabled
+automatically with a C<use v7> (or higher) declaration in the current
```

Comment:
Instead of having the function docs have to say what feature bundles are relevant, this should link to `L<feature/The 'trim' feature>` which would have these details.

Reply:
I changed this to link to the trim feature page... I copied it from the "say" documentation